build.sh

```diff
@@ -4,9 +4,13 @@ if [ -z "$1" ]
   then
     echo "Bulding with Dockerfile.ssh ..."
     docker build -t ompi-toy-box:latest -f Dockerfile.ssh .
-else 
+elif [ "$1" = "all" ]
+  then
+    docker build -t ompi-toy-box:latest -f Dockerfile.ssh .
+    docker build -t ompi-toy-box:slurm -f Dockerfile.slurm .
+else
     echo "Bulding with Dockerfile.$1 ..."
-    docker build -t ompi-toy-box:latest -f Dockerfile.$1 .
+    docker build -t ompi-toy-box:$1 -f Dockerfile.$1 .
```

Comment:
This is actually how this line used to look, but for some reason I had reverted it from '$1' to 'latest'.  It should not be a blocker either way; therefore, I will approve the merge.